app/templates/src/static/css/cf-theme-overrides.less

```diff
@@ -6,15 +6,21 @@
 /* cf-core
    ========================================================================== */
 
-// Sizing variables 
-
+// Sizing variables
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
 
-// Color variables 
```

Comment:
Should the `// Color variables` have been removed or just moved down below the fonts?